api/v1beta1/ironicneutronagent_types.go

```diff
@@ -0,0 +1,139 @@
+/*
+Copyright 2023.
```

Comment:
Umm... shouldn't that include who holds it? :)

Reply:
Agreed. I copied this from other files. It seems missing copyright holder is an issue in _a lot_ of places across the github.com/openstack-k8s-operators repos.

I've updated the files in this PR, but we should follow up on existing files.